scripts/increment_auto_off.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+# A program that ticks a counter down until deactivating log_persistence
+
+import requests
```

Comment:
another thing to note here. you're running with the system python, outside of any virtualenv. it looks like our default image comes with the Debian package `python3-requests`, but I don't see it in `configure.py`, which means this would otherwise break when we start using automated images. Could you pull this package in?